.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,16 @@
+blank_issues_enabled: false
+contact_links:
+ 
+  - name: Particular support
+    url: https://particular.net/support
+    about: Please contact us to discuss your support needs.
+  - name: Particular Community Support
+    url: https://discuss.particular.net/
+    about: Reach out to the particular discourse community.
+  - name: Particular consulting
+    url: https://particular.net/consulting
+    about: Looking for a quick assessment of your major architectural choices before starting a large development effort.
+  - name: Particular onsite tranining
+    url: https://particular.net/onsite-training
+    about: Please contact us for any onsite training.
```

Comment:
```suggestion
  - name:  Ask a question
    url: https://discuss.particular.net/
    about: Reach out to the particular discourse community.
  - name: Get support
    url: https://particular.net/support
    about: Please contact us to discuss your support needs.

```

Reply:
Although it's clever, in my opinion, consulting/training is too much for a GitHub repo. Also I'm suggesting ordering the options in lighter weight (Ask a question) to heavier weight  (Get support). I'm open to asking for outside opinions. I do like that this lets us link to other places.